### ISSUE TITLE
Add a CLI parameter that analyzes results but doesn't write them

### ISSUE THREAD

**Issue by Shnatsel** [japaric/criterion.rs#157]
I'm trying to get rid of some `unsafe` optimizations in a library. My workflow is as follows:

1. Benchmark existing code with `unsafe`s
1. Try a safe approach, benchmark it
1. If it's as fast as the code with `unsafe`, go to next `unsafe`; if it's slower, try another safe way to implement it.

Currently running `cargo bench` with Criterion overwrites the results, so if my safe code has regressed the baseline I'm working from will be overwritten and I'll have to revert and re-run the benchmarks. This is unnecessarily time-consuming.

Criterion currently exposes `--measure-only` CLI parameter that disables analyzing and writing results. Thing is, result analysis is the reason I'm using Criterion in the first place - I need to detect 2% changes in performance, which is comparable to variance and I just can't detect these without statistical analysis. So `--measure-only` is not an option.

Right now the best solution I've got is saving the baseline benchmark numbers in a separate folder and copying it back after every run. Needless to say, that's clunky and error-prone.

**TL;DR:** please add a CLI flag that disables writing results to disk but keeps the analysis.

**Comment by Everlag** [comment]
It sounds like #155 might address your use case. This hasn't hit a release yet but you can still use it by temporarily pinning to 4420e5adc5fca6d11aa92b6c433ba2516fe2ed9f in Cargo.toml.

**Comment by Shnatsel** [comment]
Indeed it does. Thank you!
